doc/evaluation.rst

```diff
@@ -0,0 +1,331 @@
+.. _evaluation:
+
+Evaluation metrics
+""""""""""""""""""
+
+This section docments the exact mathematical definition of the main metrics used at ETS for evaluating the performance of automated scoring engines. RSMTool reports include many additional evaluations documented in the descriptions of the :ref:`intermediary files<rsmtool_eval_files>` and the :ref:`report sections<general_sections_rsmtool>`.
+ 
+The following conventions are used in the formulas in this section:
+
+:math:`N` - total number of responses in the :ref:`evaluation set<test_file>` with numeric human score and numeric system score. Zero human scores are by default also excluded from the evaluation unless :ref:`exclude_zero_scores<exclude_zero_scores_rsmtool>` was set to ``false``.
+
+:math:`M` - system score. The primary evaluation analyses in the RSMTool report are conducted for *all* six types of :ref:`scores <score_postprocessing>`. For some additional evaluations, the user can pick between raw and scaled scores using the :ref:`use_scaled_predictions<use_scaled_predictions_rsmtool>` configuration field in RSMTool or :ref:`scale_with<scale_with_eval>` in RSMEval.
+
+:math:`H` - human score. The score values in :ref:`test_label_column<test_label_column_rsmtool>` for RSMTool or :ref:`human_score_column<human_score_column_eval>` for RSMEval.
+
+.. _h2:
+
+:math:`H2` - second human score (if available). The score values in :ref:`second_human_score_column<second_human_score_column_rsmtool>`.
+
+:math:`N_2` - total number of responses in the evaluation set where both :math:`H` and :math:`H2` are available and are numeric and non-zero (unless :ref:`exclude_zero_scores<exclude_zero_scores_rsmtool>` was set to ``false``).
```

Comment:
Two primary reasons, both related to readability: 
(1) We use `H_i` notation later to refer to `H` scores for `i` response and `H_2` would become ambiguous in this context.
(2) The formula for PRMSE is easier to read with `N_2` notation since it has a lot of `2N_2` that would need to be converted to `2*N2` making that formula very difficult to parse. 

Conceptually, the two variables are not equal, so  it makes sense to use different notation: `H2` refers to second rater, a separate measurement. `N_2` is the total number of double-scored responses, an auxiliary variable.

Reply:
Makes sense. Thanks!